### ISSUE TITLE
Incompatible with Android Gradle Plugin 8.3.0

### ISSUE THREAD

**Issue by syarihu** [DeployGate/gradle-deploygate-plugin#191]
# Environment
- Java 17
- Android Gradle Plugin 8.3.0
- Gradle 8.6
- Gradle DeployGate Plugin 2.7.0

# Outcome
The command ./gradlew tasks will fail for the following reason:

```
> Task :tasks FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tasks'.
> Failed to query the value of task ':app:uploadDeployGateDebug' property 'apkInfo'.
   > Could not get unknown property 'signingConfigData' for task ':app:packageDebug' of type com.android.build.gradle.tasks.PackageApplication.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 3s
10 actionable tasks: 3 executed, 7 up-to-date
```

**Comment by jmatsu** [comment]
We will be back when the new version would be released.

**Comment by jmatsu** [comment]
@syarihu 
We have released a fix in v2.8.0. Appreciate your report.
